src/api/contract/configuration/v1.ts

```diff
@@ -0,0 +1,23 @@
+// This module is contractual and should not be changed after release.
+// It should be in sync with vscode-kubernetes-tools-api/ts/configuration/v1.ts
+// at all times.
+
+export interface ConfigurationV1 {
```

Comment:
You will need to update `api/apibroker.ts` to expose the new interface category on the extension API object.  `apibroker` is the secret handshake between a consumer of the extension API and the various interface implementations - I'm afraid it's not very visible and I should have flagged it when we first talked about making a new interface category.  Sorry!

Reply:
Ahh that makes sense! Thanks for pointing that out